src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java

```diff
@@ -79,10 +106,17 @@ public void theResponseStatusCodeShouldBeOnPatchAccountRequest(final String stat
 
     @When("I execute the delete Account request on {string} endpoint")
     public void iExecuteTheDeleteAccountRequestOnEndpoint(final String endpoint) {
+        requestBuilder
```

Comment:
parameter not used. It should be used or deleted.

Reply:
Thanks, I have just fixed it